test/DynamoCoreWpfTests/PackageManagerUITests.cs

```diff
@@ -233,6 +254,27 @@ public void PackageSearchDialogSearchBoxPromptTextWhenNotSyncing()
             Assert.AreEqual(Dynamo.Wpf.Properties.Resources.PackageSearchViewSearchTextBox, searchViewModel.SearchBoxPrompt);
         }
 
+        [Test]
+        public void PackageManagerCrashTestOnDownloadingInvalidPackage()
+        {
+            string packageDirectory = Path.Combine(GetTestDirectory(ExecutingDirectory), @"core\PackageManagerTests\Autodesk Steel Package");
+
+            try
+            {
+                LoadPackage(packageDirectory);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Failed to load the package: "+ e);
+            }
+
+            var loader = GetPackageLoader();
+            Assert.AreEqual(loader.LocalPackages.Count(), 0);
```

Comment:
Asserting that the actually packages count doesn't increase and making sure that the Manage Packages window shows up.